src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
```

Comment:
small improvement: if doughnutType was an enum, we wouldn't have to worry about the IllegalArgumentException (although presumably we'd somewhere have to create that enum from a user-provided string, but that logic probably isn't this class' responsibility)